webui/src/lib/auth/authContext.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, useContext, useMemo, useState, ReactNode } from "react";
+import { AUTH_STATUS, type AuthStatus } from "./status";
+
+type AuthContextShape = {
```

Comment:
What's "shape"?
Let's align to `AuthContextType` like the other contexts we have.